packages/devextreme/scss/widgets/fluent/_colors.scss

```diff
@@ -141,7 +141,8 @@ $base-inverted-icon-color: null !default;
 $base-disabled-opacity: 0.38 !default;
 $base-disabled-color: null !default;
 $base-border-color-disabled: null !default;
-$base-dropdown-shadow-color: null !default;
+$base-dropdown-shadow-color1: null !default;
```

Comment:
This file should contain only basic definitions of variables without specifying the widgets to which they relate. If variables are needed for dropdown, then they must be in the appropriate files.
We are already 'base-shadow-color' and it will be better to get the necessary transparency from this color in the final places where it should be used via color.change($base-shadow-color, $alpha: ...) function